environment.go

```diff
@@ -200,6 +205,19 @@ func (e *envState) createQueues(ctx context.Context) error {
 	}
 	e.remoteQueue = rq
 
+	// Notifications queue w/ moving weight avg pool
+	e.notificationsQueue = queue.NewLocalLimitedSize(e.settings.Amboy.PoolSizeLocal, e.settings.Amboy.LocalStorage)
```

Comment:
the pool size should be way smaller here, maybe len(senders)?

Reply:
done, had to move sender initialisation to before the queue.